### PR TITLE
Reduce debug on domain connection

### PR DESCRIPTION
**Pull request by Atlante45**
Removes/Reduces some debug I noticed slow down domain connection/reconnection

### CODE REVIEW

File: libraries/render-utils/src/Model.cpp [hunk 1]
```diff
@@ -913,8 +913,6 @@ bool Model::addToScene(std::shared_ptr<render::Scene> scene, render::PendingChan
 
     bool somethingAdded = false;
 
-    qDebug() << "Model::addToScene : " << this->getURL().toString();
```
Comment (Atlante45) [inline libraries/render-utils/src/Model.cpp]: @samcake is it cool to remove those?

File: libraries/render-utils/src/Model.h [hunk 2]
```diff
@@ -312,8 +312,7 @@ class Model : public QObject, public PhysicsEntity {
     float getLimbLength(int jointIndex) const;
     
     /// Allow sub classes to force invalidating the bboxes
-    void invalidCalculatedMeshBoxes() { 
-        qDebug() << "invalidCalculatedMeshBoxes()";
```
Comment (Atlante45) [inline libraries/render-utils/src/Model.h]: @ZappoMan Do you want to keep that?

**Comment by hifi-gustavo** [comment]
Build console output for commit: 90c9a82aeb889423c65de73d42025c6f6b30b31a available at: https://gist.github.com/35c1e92936e624b0136c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5087.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5087.dmg

File: libraries/render-utils/src/Model.cpp (hunk 1 above)
Comment (samcake) [inline libraries/render-utils/src/Model.cpp]: Sure

Comment (ZappoMan) [inline libraries/render-utils/src/Model.cpp]: ok with me

File: libraries/render-utils/src/Model.h (hunk 2 above)
Comment (ZappoMan) [inline libraries/render-utils/src/Model.h]: ok with me

**Comment by hifi-gustavo** [comment]
Build console output for commit: bf56ee49a63807e7dd1193830ff05336b3972899 available at: https://gist.github.com/35c1e92936e624b0136c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5087.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5087.dmg

**Pull request merged**